src/i18n/translations/en.ts

```diff
@@ -650,4 +650,5 @@ export const messages: MessagesKey = {
   mapLoadingStopsDataText: 'Loading stops...',
   locateStopPoint: 'Locate',
   locateStopPointTooltip: 'Locate on map',
+  dropdownNoMatchesText: 'No options available',
```

Comment:
Swedish: Inga val tillgängliga

Reply:
Finnish: _Ei vaihtoehtoja_ or _Ei valintoja_

Former is literally "No options", which is grammatically correct but sounds very absolute in Finnish. Latter is literally "No choices", which IMHO fits better, even though the English counterpart is harsher than for the other one.